src/ucp/api/ucp_def.h

```diff
@@ -87,6 +87,29 @@ typedef struct ucp_address               ucp_address_t;
 
 
 /**
+ * @ingroup UCP_ENDPOINT
+ * @brief Error handling mode for the UCP endpoint.
+ * 
+ * Specifies error handling mode for the UCP endpoint.
+ */
+typedef enum {
+    UCP_ERR_HANDLING_MODE_DEFAULT,          /**< Propagates only locally
+                                             *   detected errors, provides
+                                             *   minimal overhead from
+                                             *   perfromance perspective */
+    UCP_ERR_HANDLING_MODE_PEER              /**< In addition to @ref
+                                             *   UCP_ERR_HANDLING_MODE_DEFAULT
+                                             *   propagates errors also related
```

Comment:
"UCP_ERR_HANDLING_MODE_DEFAULT propagates errors also related with" ->
"UCP_ERR_HANDLING_MODE_DEFAULT, also propagates errors related to"